### ISSUE TITLE
MultiPersistent broken on 6.99.12

### ISSUE THREAD

**Issue by Ria-kon** [renpy/renpy#1095]
I just reinstall RenPy to 6.99.12 from 6.99.11 and than i take only one issue with MP. Everything work on older version, but with newest version MP just seems like something not working. 
After playing with reencoding utf-8 script file and even separating code i just create new one project and paste renpy example in new file:
```
init python:
    mp = MultiPersistent("demo.renpy.org")
```
And hooray, even this not working!
```
While running game code:
  File "game/1.rpy", line 2, in <module>
    mp = MultiPersistent("demo.renpy.org")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc0 in position 9: ordinal not in range(128)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "lib/windows-i686/1.rpyc", line 1, in script
  File "C:\renpy-6.99.12-sdk\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\renpy-6.99.12-sdk\renpy\python.py", line 1695, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/1.rpy", line 2, in <module>
    mp = MultiPersistent("demo.renpy.org")
  File "C:\renpy-6.99.12-sdk\renpy\persistent.py", line 469, in MultiPersistent
    fn = fn + "/" + name
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc0 in position 9: ordinal not in range(128)
```
And i attached test project with log, traceback and 1.rpy (just with multipersistent function) and everything else
[1.zip](https://github.com/renpy/renpy/files/667334/1.zip)

**Comment by renpytom** [comment]
I'm fixing this in Ren'Py ASAP, but as a workaround for now, write:

init python:
    mp = MultiPersistent(b"demo.renpy.org")

**Comment by renpytom** [comment]
Fixed in 6.99.12.1, which was released a few days ago.
